### ISSUE TITLE
clear interval when new interval is set

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/hoodie-client-connection-status#3]
the current code only clears the interval if `connectionStatus.check()` is called without `interval: ` option. But it should always be cleared, otherwise we will have two check intervals running in parallel, and we lost the pointer for the first one

**Comment by gr2m** [comment]
closing in favor of #13
